### PR TITLE
coprocessor: support default value.

### PR DESCRIPTION
**Pull request by coocood**
When default value is provided in column info, use that value if not found.

### CODE REVIEW

**Comment by coocood** [comment]
@BusyJay @AndreMouche PTAL

File: tests/coprocessor/test_select.rs [hunk 1]
```diff
@@ -89,18 +89,21 @@ struct Column {
     col_type: i32,
     // negative means not a index key, 0 means primary key, positive means normal index key.
     index: i64,
+    default_val: Option<i64>,
```
Comment (AndreMouche) [inline tests/coprocessor/test_select.rs]: why not use `Option<Vec<u8>>` ?

Comment (coocood) [inline tests/coprocessor/test_select.rs]: Vec<u8> doesn't implement Copy trait.

Comment (BusyJay) [inline tests/coprocessor/test_select.rs]: I prefer `Option<Vec<u8>>` too.

File: src/server/coprocessor/endpoint.rs [hunk 2]
```diff
@@ -803,6 +806,8 @@ impl SelectContextCore {
             }
             if col.get_pk_handle() {
                 box_try!(datum::encode_to(chunk.mut_rows_data(), &[get_pk(col, h)], false));
+            } else if col.has_default_val() {
```
Comment (BusyJay) [inline src/server/coprocessor/endpoint.rs]: Can it occur in aggregation or top-n?

Comment (coocood) [inline src/server/coprocessor/endpoint.rs]: No for the current implementation.
 If the row doesn't contain the column, and it has default value it's not in public state.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**